instructure/instructure-ui
# INSTUI-4188 New Table API to allow custom children

- [x] use Context instead of passing down props
- [x] Allow custom children
- [x] write test for custom children
- [x] make sure that one can use falsey/null children
- [x] write example and docs for the new API

To test:

- Check out the new examples and play with them.   
- Try to add custom components as children that just wrap existing components.               
- Try to make fully new custom components.  
- Add stupid things as children to components (see one of the new tests), it should not result in an exception
